components/ColumnComponents/PercentContainer/PercentContainer.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
-import PiePercent from '../../PiePercent';
+// import PiePercent from '../../PiePercent';
 import styles from './PercentContainer.css';
+import LinearProgress from 'material-ui/LinearProgress';
 
 const getPercentWin = (wins, games) => (games ? Math.ceil(1000 * (wins / games)) / 10 : 0);
```

Comment:
ok I am not putting toFixed inside the function because I think we should only display 1 decimal point for some things, so we can toFixed the result of that output.